lib/app/utils/hl7_validator.rb

```diff
@@ -25,10 +27,31 @@ def validate(resource, fhir_models_klass, profile_url = nil)
       information = issues_by_severity(outcome.issue, 'information')
       {
         errors: fatals.concat(errors).reject(&:empty?),
-        warnings: warnings.concat(information).reject(&:empty?)
+        warnings: warnings,
+        information: information
```

Comment:
I think we were intentionally not including informational messages here.

Reply:
I think we do want them here and this will make the functionality closer to the [recent code in program](https://github.com/onc-healthit/inferno-program/blob/b9779b708e2262dd21f129bbf7b9f8740e0e3d69/lib/app/utils/hl7_validator.rb#L29) where information messages are not lumped in with warnings.

I don't think they are actually used anywhere, but adding them here at least makes them available to tests and we won't be throwing warnings unexpectedly.

I vaguely remember there was a PR somewhere to use information messages and it included a little `i` icon, but I can't find it.

Reply:
Could be wrong though

Reply:
In `inferno-program`, we do support the gathering of `@information_messages` in `sequence_base.rb`, and we add any `information` messages from the validator into that array.

Reply:
I can't actually find where we display them, though.

Reply:
I think we removed them because we decided they weren't something we wanted to display in program.

Reply:
here was the PR with the little `i`.  Looks like in the last commit we decided not to display the informational messages separately.

I still think this change is worthwhile though so that we don't show information messages as warnings.